src/languages/html.coffee

```diff
@@ -0,0 +1,681 @@
+helper = require '../helper.coffee'
+parser = require '../parser.coffee'
+
+parse5 = require '../../vendor/parse5/index.js'
```

Comment:
https://github.com/inikulin/parse5/compare/master...sakagg:master

I've forked the repo and I can add the link to this fork in packages.json.